doc/install/manual-deployment.rst

```diff
@@ -348,116 +348,70 @@ Long Form
 
 Without the benefit of any helper utilities, create an OSD and add it to the
 cluster and CRUSH map with the following procedure. To create the first two
-OSDs with the long form procedure, execute the following on ``node2`` and
-``node3``:
+OSDs with the long form procedure, execute the following steps for each OSD.
 
-#. Connect to the OSD host. :: 
+.. note:: This procedure does not describe deployment on top of dm-crypt
+          making use of the dm-crypt 'lockbox'.
 
-	ssh {node-name}
-
-#. Generate a UUID for the OSD. ::
-
-	uuidgen
+#. Connect to the OSD host and become root. ::
 
+     ssh {node-name}
+     sudo bash
 
-#. Create the OSD. If no UUID is given, it will be set automatically when the 
-   OSD starts up. The following command will output the OSD number, which you 
-   will need for subsequent steps. ::
-	
-	ceph osd create [{uuid} [{id}]]
-
-
-#. Create the default directory on your new OSD. :: 
-
-	ssh {new-osd-host}
-	sudo mkdir /var/lib/ceph/osd/{cluster-name}-{osd-number}
-	
+#. Generate a UUID for the OSD. ::
 
-#. If the OSD is for a drive other than the OS drive, prepare it 
-   for use with Ceph, and mount it to the directory you just created:: 
+     UUID=$(uuidgen)
 
-	ssh {new-osd-host}
-	sudo mkfs -t {fstype} /dev/{hdd}
-	sudo mount -o user_xattr /dev/{hdd} /var/lib/ceph/osd/{cluster-name}-{osd-number}
+#. Generate a cephx key for the OSD. ::
 
-	
-#. Initialize the OSD data directory. :: 
+     OSD_SECRET=$(ceph-authtool --gen-print-key)
```

Comment:
there is an alternative to `--gen-print-key` right? or is this the only way of achieving this